bundles/org.openhab.binding.tibber/README.md

```diff
@@ -98,189 +43,425 @@ Note: Tibber HomeId is retrieved from [developer.tibber.com](https://developer.t
+| daily-consumption     | Number:Energy             | Energy consumed since midnight in kilowatt-hours              |
+| daily-cost            | Number:Currency           | Accumulated cost since midnight                               |
+| last-hour-consumption | Number:Energy             | Energy consumed since last hour shift in kilowatt-hours       |
+| total-production      | Number:Energy             | Total energy production measured by Tibber Pulse meter        |
+| daily-production      | Number:Energy             | Net energy produced since midnight in kilowatt-hours          |
+| last-hour-production  | Number:Energy             | Net energy produced since last hour shift in kilowatt-hours   |
+
+## Thing Actions
+
+Thing actions can be used to perform calculations on the current available price information cached by the binding. 
+Cache contains energy prices from today and after reaching the `updateHour` also for tomorrow.
+This is for planning when and for what cost a specific electric consumer can be started.  
+
+Performing a calcuation a `parameters` object is needed containing e.g. your boundaries for the calculation.
+Parameter object allow 2 types: Java `Map` or JSON `String`.
+The result is returned as JSON encoded `String`.
+Refer below sections how the result looks like.
+Some real life scenarios are schown in [Action Examples](#action-examples) section.
 
-When Tibber Binding is installed, Tibber API should be auto discovered.
+### `priceInfoStart`
 
-Retrieve personal token and HomeId from description above, and initialize/start a scan with the binding.
+Returns starting point as `Instant` of first available energy price. 
+It's not allowed to start calculations before this timestamp.
 
-Tibber API will be auto discovered if provided input is correct.
+In case of error `Instant.MAX` is returned.
 
-## Tomorrow and Today Prices
+### `priceInfoEnd`
 
-The today and tomorrow prices are served as forecast on the `current_total` channel and as JSON data on the channels `today_prices` and `tomorrow_prices`.
-Example of tomorrow and today prices data structure - an array of tuples:
+Returns end point as `Instant` of the last available energy price. 
+It's not allowed to exceed calculations after this timestamp.
+
+In case of error `Instant.MIN` is returned.
+
+### `listPrices`
+
+List prices in ascending / decending _price_ order.
+
+#### Parameters
+
+| Name          | Type      | Description                           | Default           | Required  |
+|---------------|-----------|---------------------------------------|-------------------|-----------|
+| earliestStart | Instant   | Earliest start time                   | now               | no        |
+| latestStop    | Instant   | Latest end time                       | `priceInfoEnd`    | no        |
+| ascending     | boolean   | Hour when spot prices are updated     | true              | no        |
```

Comment:
Description doesn't match parameter. And how would `asecending: false` sort? By `earliestStart`or `latestStop`? 

Suggestion: Use a constant/enum for which field to sort on, ie `PRICE_ASCENDING`, `PRICE_DESCENDING`, `EARLIEST_START_ASCENDING`and so on. Or omit parameter and let the user handle sorting.

I tried `ascending:false` but I couldn't make it sort by time (could be user error, did not check binding code)

Reply:
Yes, description is wrong - copy & paste :(

Ordering is according to _prices_ as described
<img width="275" height="65" alt="image" src="https://github.com/user-attachments/assets/d1f48359-84a8-415d-b12c-e8ed3a3e1adc" />

For _time_ there are already [default rules available](https://www.openhab.org/docs/configuration/persistence.html#persistence-extensions-in-scripts-and-rules) like `<item>.getAllStatesBetween(ZonedDateTime, ZonedDateTime)`